packages/commonwealth/test/util/evm-chain-testing/src/routes/erc1155.ts

```diff
@@ -31,7 +31,7 @@ export const deploy1155 = async (req: Request, res: Response) => {
     const account = (await provider.eth.getAccounts())[0];
     await contract
       .deploy({ data: testErc1155Bytecode })
-      .send({ from: account, gas: 5_000_000 })
+      .send({ from: account, gas: '5000000' })
```

Comment:
You didn't upgrade Web3 in the `evm-chain-testing/package.json` so checking types here throws an error. Honestly in a future iteration we should consider moving this package out of `/utils` and into `libs` since we likely will want to use it for other tests elsewhere (contests, stake, etc). Maybe as part of the work to switch to `anvil.js`.

Reply:
updated the package here, lmk if it looks like I missed anything there

Reply:
The `yarn start` command for the package still fails for me due to type errors. Add `"check-types": "tsc --noEmit",` to the `package.json` scripts and execute it to find the issues (leave the script in for the future). I believe we need to rebuild the Docker image and push it up to the Docker repo as well.

Reply:
added